### PR TITLE
TELUS.BusinessConnectPhone version 23.1.1.45971

### PR DESCRIPTION
**Pull request by VA6DAH**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/101468)

### CODE REVIEW

**Comment by VA6DAH** [comment]
@microsoft-github-policy-service agree

File: manifests/t/TELUS/TELUSBusinessConnectPhone/23.1.1.45971/TELUS.TELUSBusinessConnectPhone.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: TELUS.TELUSBusinessConnectPhone
```
Comment (Trenly) [inline manifests/t/TELUS/TELUSBusinessConnectPhone/23.1.1.45971/TELUS.TELUSBusinessConnectPhone.installer.yaml:4]:
```suggestion
PackageIdentifier: TELUS.BusinessConnectPhone
```
Best practice is to not repeat the publisher. After making this change, the folder and file names will need to be updated also

File: manifests/t/TELUS/TELUSBusinessConnectPhone/23.1.1.45971/TELUS.TELUSBusinessConnectPhone.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: TELUS.TELUSBusinessConnectPhone
```
Comment (Trenly) [inline manifests/t/TELUS/TELUSBusinessConnectPhone/23.1.1.45971/TELUS.TELUSBusinessConnectPhone.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: TELUS.BusinessConnectPhone
```
Best practice is to not repeat the publisher. After making this change, the folder and file names will need to be updated also

File: manifests/t/TELUS/TELUSBusinessConnectPhone/23.1.1.45971/TELUS.TELUSBusinessConnectPhone.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: TELUS.TELUSBusinessConnectPhone
```
Comment (Trenly) [inline manifests/t/TELUS/TELUSBusinessConnectPhone/23.1.1.45971/TELUS.TELUSBusinessConnectPhone.yaml:4]:
```suggestion
PackageIdentifier: TELUS.BusinessConnectPhone
```
Best practice is to not repeat the publisher. After making this change, the folder and file names will need to be updated also

File: manifests/t/TELUS/TELUSBusinessConnectPhone/23.1.1.45971/TELUS.TELUSBusinessConnectPhone.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: TELUS.TELUSBusinessConnectPhone
+PackageVersion: 23.1.1.45971
+PackageLocale: en-US
+Publisher: TELUS
+PackageName: TELUS Business Connect Phone
+License: http://telus.com/BusinessConnect/ServiceTerms
```
Comment (Trenly) [inline manifests/t/TELUS/TELUSBusinessConnectPhone/23.1.1.45971/TELUS.TELUSBusinessConnectPhone.locale.en-US.yaml:9]:
```suggestion
License: Proprietary
LicenseUrl: http://telus.com/BusinessConnect/ServiceTerms
```
A URL is not a License

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by stephengillie — APPROVED** [review]

**Pull request merged**